### ISSUE TITLE
How to obtain the pretraining data?

### ISSUE THREAD

**Issue by zoey731** [lvkd84/GraphFP#2]
Thanks for your great jobs! I noticed your paper mentioned "We use a processed subset containing 456K molecules from the ChEMBL database [24] for pretraining." Could you please release your pretraining data or give detailed instructions how to obtain it. Thanks!

**Comment by lvkd84** [comment]
Hi,
I've updated the README with this detail.
Essentially, you can access the prepared dataset for pretraining [here](https://drive.google.com/file/d/17QGpb3V3kQIDnqSraGBXYqt7KDVxXsEw). 
However, if you have a different torch-geometric version than the one we used then it may not work. In such case, you can prepare the dataset yourself from the raw SMILES strings. To do so, download and unzip the data from [chem data](https://snap.stanford.edu/gnn-pretrain/data/chem_dataset.zip). Navigate to `dataset/chembl_filtered/processed/smiles.csv`. This file should contains the SMILES strings of all the molecules used in pretraining.
Hopefully I didn't miss anything. Let me know if you need more help.

**Comment by wjxts** [comment]
Based on dataset/chembl_filtered/processed/smiles.csv, should I use mol/prepare_data_old.py to preprocess the dataset?

**Comment by lvkd84** [comment]
> Based on dataset/chembl_filtered/processed/smiles.csv, should I use mol/prepare_data_old.py to preprocess the dataset?

Yes. Actually, I have just commited this file as mol/chembl_pretraining_data.csv. If you want to prepare this training set, you can run:

`python prepare_data_old --root <root path> --data_file_path chembl_pretraining_data.csv --smiles_column smiles --vocab_file_path vocab.txt `

**Comment by sherlockma11** [comment]
Thanks for your great jobs!

**Comment by sherlockma11** [comment]
Thanks for your great jobs!
If I want to pretrain the model, I should use mol/prepare_data_old.py to process the dataset/chembl_filtered/processed/smiles.csv,then use train_gnn_prepredictive_old.py to pretrain the model.
If I want to use my pretrained model to train downstream task like HIV,I should use downstream_old.py
Is it right?

**Comment by lvkd84** [comment]
@sherlockma11 Yes, that is correct.
